examples/data_pipeline/roberta_dataframe.py

```diff
@@ -0,0 +1,86 @@
+from argparse import ArgumentParser
+from functools import partial
+
+import torcharrow as ta
+import torcharrow.dtypes as dt
+import torcharrow.pytorch as tap
+import torchtext.functional as F
+import torchtext.transforms as T
+from torch.hub import load_state_dict_from_url
+from torch.nn import Module
+from torch.utils.data import DataLoader
+from torchtext.datasets import SST2
+
+
+class RobertaTransform(Module):
+    def __init__(self) -> None:
+        super().__init__()
+        # Instantiate various transforms
+
+        # Tokenizer to split input text into tokens
+        encoder_json_path = "https://download.pytorch.org/models/text/gpt2_bpe_encoder.json"
+        vocab_bpe_path = "https://download.pytorch.org/models/text/gpt2_bpe_vocab.bpe"
+        self.tokenizer = T.GPT2BPETokenizer(encoder_json_path, vocab_bpe_path)
+
+        # vocabulary converting tokens to IDs
+        vocab_path = "https://download.pytorch.org/models/text/roberta.vocab.pt"
+        self.vocab = T.VocabTransform(load_state_dict_from_url(vocab_path))
+
+        # Add BOS token to the beginning of sentence
+        self.add_bos = T.AddToken(token=0, begin=True)
+
+        # Add EOS token to the end of sentence
+        self.add_eos = T.AddToken(token=2, begin=False)
+
+    def forward(self, input: ta.DataFrame) -> ta.DataFrame:
+        input["tokens"] = input["text"].map(self.tokenizer, dtype=dt.List(dt.string))
+        input["tokens"] = input["tokens"].map(partial(F.truncate, max_seq_len=254))
```

Comment:
nit: is this try to reserve the first 254 elements in `tokens`? try 

```python
input["tokens"].list.slice(stop=254)
```

Reply:
yupp, good call. BTW, do we have append/insert operations available? that way we can use that instead of explicitly calling transforms for adding tokens ids at the begin/end as done in below using self.add_bos?

Reply:
Didn't find it in https://facebookincubator.github.io/velox/functions/array.html. But array concat is definitely a function in Presto: https://prestodb.io/docs/current/functions/array.html. Will ask Velox developers on this. 

Just want to check the semantic of `add_bos` -- basically we want to add one element at the beginning or end of the array right?